### PR TITLE
Facebook list

### PR DESCRIPTION
**Pull request by LukeP91**

### CODE REVIEW

File: app/assets/javascripts/send_to_facebook.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+$(document).ready(() => {
+  console.log('runned', $(".send-to-facebook"))
```
Comment (simon2k) [inline app/assets/javascripts/send_to_facebook.js]: ✂️  - remove all `console.log`

File: app/assets/javascripts/send_to_facebook.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  $(".send-to-facebook").click((e) => {
+      console.log('click')
+      e.preventDefault();
+      const tweetId = $(e.target).data().tweetId;
+      $.post(`/admin/facebook_posts/${tweetId}/send_to_facebook`)
+        .done(() => {
+          $( ".main-container" ).prepend( "<div class='alert alert-success'>Tweet has been send to Facebook</div>" );
```
Comment (simon2k) [inline app/assets/javascripts/send_to_facebook.js]: remove spaces before and after curly brackets - what's with code autoformation?

File: app/controllers/admin/facebook_posts_controller.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
+class Admin::FacebookPostsController < ApplicationController
+  def index
+    tweets = Tweet.in_organization(current_organization).not_send_to_facebook
+    authorize tweets
+    render :index, locals: { tweets: tweets }
+  end
+
+  def send_to_facebook
+    SendTweetToFacebook.call(user: current_user, tweet_id: params[:id], organization: current_organization)
```
Comment (simon2k) [inline app/controllers/admin/facebook_posts_controller.rb:9]: make this call async and updating listing using action cable, when post was posted to facebook (a new idea ✌️)

File: config/routes.rb [hunk 4]
```diff
@@ -30,6 +30,11 @@
     resources :categories
     resources :interests
     resources :sources, only: %i[index new create destroy]
+    resources :facebook_posts, only: :index do
+      member do
+        post 'send_to_facebook'
```
Comment (simon2k) [inline config/routes.rb]: use symbol

File: spec/features/admin/twitter/facebook_posts/admin_facebook_posts_index_spec.rb [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+      app.home_page.menu.admin_facebook_posts_link.click
+      expect(app.admin_facebook_posts_index_page).to be_displayed
+
+      app.admin_facebook_posts_index_page.send_button(tweet.id).click
+      expect(facebook_wrapper).to have_received(:post_on_wall).with('New blog post is up #rails #ruby')
+    end
+  end
+end
+
+
```
Comment (simon2k) [inline spec/features/admin/twitter/facebook_posts/admin_facebook_posts_index_spec.rb]: ✂️  - does trimming blank lines and trailing spaces work?

File: app/views/admin/sources/index.html.haml [hunk 6]
```diff
@@ -8,10 +8,11 @@
...
     - sources.each do |source|
-      %tr
-        %td= source.name
-        %td
-          = link_to t('.destroy', default: t("helpers.links.destroy")), admin_source_path(source), method: :delete, data: { confirm: t('.confirm', default: t("helpers.links.confirm", default: 'Are you sure?')) }, class: 'btn btn-xs btn-danger', id: "source_delete_#{source.id}"
+      - cache source do
+        %tr
+          %td= source.name
+          %td
+            = link_to t('.destroy', default: t("helpers.links.destroy")), admin_source_path(source), method: :delete, data: { confirm: t('.confirm', default: t("helpers.links.confirm", default: 'Are you sure?')) }, class: 'btn btn-xs btn-danger', id: "source_delete_#{source.id}"
```
Comment (simon2k) [inline app/views/admin/sources/index.html.haml]: time to split this line to multiple lines

**Review by simon2k — CHANGES_REQUESTED** [review]

File: app/controllers/admin/facebook_posts_controller.rb (hunk 3 above)
Comment (LukeP91) [reply app/controllers/admin/facebook_posts_controller.rb:9]: Will be changed in next PR

**Review by simon2k — APPROVED** [review]

**Pull request merged**